app/scripts/controllers/UsernamePromptCtrl.js

```diff
@@ -0,0 +1,22 @@
+(function(){
+
+  function UsernamePromptCtrl($uibModalInstance, $cookieStore) {
+
+
+    this.submit = function() {
+      if(this.text) {
+
+        $cookieStore.put('blocChatCurrentUser', this.text);
```

Comment:
Where are you planning on storing the current user in your application state?

This will add it to the cookie so it is there when you reload, but does not add it anywhere in your application. If you plan on using `BlocChatCookies` service, you may want to add a set method there, and while you're at it.. have to do the get/set interaction with the cookie